components/style/servo/selector_parser.rs

```diff
@@ -29,7 +29,7 @@ use style_traits::{ParseError, StyleParseErrorKind};
 /// A pseudo-element, both public and private.
 ///
 /// NB: If you add to this list, be sure to update `each_simple_pseudo_element` too.
-#[derive(Clone, Debug, Eq, Hash, MallocSizeOf, PartialEq, ToShmem)]
+#[derive(Clone, Copy, Debug, Deserialize, Eq, Hash, MallocSizeOf, PartialEq, Serialize, ToShmem)]
```

Comment:
I don't _think_ you're using the `Copy` here, but ICBW.

Reply:
I just double-checked. For some reason this is necessary in a few places.